draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -539,25 +585,26 @@ following fields:
 
 * Number: Wire format suberror code (range 0-255)
 
-* Meaning: A short description of the error
+* Meaning: A short description of the sub-error
 
-* Applicable EDO Codes: List the Extended DNS Error codes to which the suberror code applies.
+* Applicability: Indicates which RFC8914 error codes apply to this sub-error code
 
-* Reference: A pointer to the specification text
+* Reference: A pointer to IETF-approved specification
 
 * Change Controller: Person or entity, with contact information if appropriate.
 
 The SubError Code registry shall initially be populated with the
 following suberror codes:
 
-| Number | Meaning | Applicable EDO Codes| Reference |  Change Controller |
-| 0 | Reserved | N/A| This-Document | IETF |
-| 1 | Malware | "Censored", "Filtered", "Blocked", and "Forged" |This-Document | IETF |
-| 2 | Phishing | "Censored", "Filtered", "Blocked", and "Forged" |This-Document | IETF |
-| 3 | Spam  | "Censored", "Filtered", "Blocked", and "Forged" |This-Document | IETF |
-| 4 | Spyware | "Censored", "Filtered", "Blocked", and "Forged" |This-Document | IETF |
-| 5 | Adware | "Censored", "Filtered", "Blocked", and "Forged" |This-Document | IETF |
-| 6 | Network policy imposed by the operator of the network | "Blocked", and "Forged" |This-Document | IETF |
+| Number | Meaning | RFC8914 error code applicability | Reference |  Change Controller |
+|:------:|:--------|:---------------------------------|:----------|:------------------:|
+| 0 | Reserved| Blocked, Forged, Censored, Filtered | {{policy-reserved}} | IETF |
```

Comment:
I would also list this document under Reference to keep track of documents that registered a code.